### PR TITLE
Backup Server

### PR DESCRIPTION
**Pull request by ayushyamitabh**
**Time Spent:** ~4 hours

**External IP:** http://35.211.175.158:8080/
**Server IP:** http://35.211.175.158:5000/

### How It Works:

1. Database is fetched every hour and stored as `database.json`.
    - Size is not an issue for our particular case - but could be broken into sub files of 100 MB each in later versions
    - If old backup exists, delete it and make new backup
    - Logger takes note of all events, along with time taken to backup.
2. Local API endpoints provided:
    - POST - '/' : just to check server status
    - POST - '/logs': get logged events
    - POST - '/backup': get backed up version of database
    - POST - '/verifyuser': used to verify super user on login
3. Result of each API is also logged

### CODE REVIEW

**Comment by hetobias** [comment]
Great job, only one nit picky thing that @khelif96 will also have a problem with. Newline at the end of the files please. Otherwise, approved.

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
-*/node_modules
+*/node_modules/*
```
Comment (khelif96) [inline .gitignore]: Shouldn't it ignore  node_modules in any directory by default?

Comment (ayushyamitabh) [reply .gitignore]: it was buggin out that seemed to fix it tho

File: backup-server/watcher/src/Google-Sans/demo.html [hunk 2]
```diff
@@ -0,0 +1,337 @@
+<!DOCTYPE html>
```
Comment (khelif96) [inline backup-server/watcher/src/Google-Sans/demo.html]: This file isnt necessary can you delete it.

**Comment by khelif96** [comment]
image
Text gets cut off at the end.

**Pull request merged**